js/packages/cli/src/commands/verifyTokenMetadata/token-metadata.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "Token Metadata",
```

Comment:
this schema is non-exhaustive (doesn't capture things like `attributes`) but does validate against the critical properties in the config